lib/Views/src/layouts/DateTimeView32x8.h

```diff
@@ -177,6 +178,26 @@ class DateTimeView32x8 : public IDateTimeView
         m_textWidget.setFormatStr(formatText);
     }
 
+    /**
+     * Get the start of week configuration (Sunday = 0).
+     *
+     * @return uint8_t
+     */
+    uint8_t getStartOfWeek() const override
+    {
+        return m_startOfWeek;
+    }
+
+    /**
+     * Set the start of week configuration (Sunday = 0).
+     *
+     * @param[in] startOfWeek uint8_t offset for starting of week
+     */
+    void setStartOfWeek(uint8_t startOfWeek) override
+    {
+        m_startOfWeek = startOfWeek;

```

Comment:
Please ensure that the startOfWeek parameter is not greater or equal than MAX_LAMPS.